Packs/PrismaCloud/LayoutRules/Prisma_Cloud_V2.json

```diff
@@ -0,0 +1,68 @@
+{
+  "rule_id": "Prisma_Cloud_V2_Layout_Rule",
+  "layout_id": "Prisma Cloud V2",
+  "description": "display for Prisma Cloud alerts.",
+  "rule_name": "Prisma Cloud V2 Layout Rule",
+  "alerts_filter": {
+    "filter": {
+      "OR": [
+        {
+            "SEARCH_FIELD": "alert_type",
```

Comment:
We need to make sure that "CONTAINS" supported in alert type filter. Otherwise, change to "EQ"